flytestdlib/resolver/k8s_resolver.go

```diff
@@ -0,0 +1,179 @@
+package resolver
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"strings"
+	"sync"
+
+	"google.golang.org/grpc/grpclog"
+	"google.golang.org/grpc/resolver"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/flyteorg/flyte/flytestdlib/logger"
+)
+
+const (
+	Schema = "k8s"
```

Comment:
I think you were going to rename it to 
```suggestion
	Scheme = "k8s"
```

Reply:
Yeah that's what I meant.